lib/runCode.js

```diff
@@ -162,12 +162,11 @@ module.exports = function (opts, cb) {
        * @property {String} opcode the next opcode to be ran
        * @property {BN} gasLeft amount of gasLeft
        * @property {Array} stack an `Array` of `Buffers` containing the stack
-       * @property {Trie} storageTrie the storage [trie](https://github.com/wanderer/merkle-patricia-tree) for the account
        * @property {Account} account the [`Account`](https://github.com/ethereum/ethereumjs-account) which owns the code running
        * @property {Buffer} address the address of the `account`
        * @property {Number} depth the current number of calls deep the contract is
        * @property {Buffer} memory the memory of the VM as a `buffer`
-       * @property {FunctionalRedBlackTree} cache the account cache. Contains all the accounts loaded from the trie. It is an instance of [functional red black tree](https://www.npmjs.com/package/functional-red-black-tree)
+       * @property {StateManager} storageManager a state manager instance (EXPERIMENTAL - unstable API)
```

Comment:
Good that this aligned documentation is already used and further evolved and corrected, really glad that I once went through this hazzle, even if there might be still some glizzles every here and there to be corrected.